nemo/collections/tts/helpers/helpers.py

```diff
@@ -204,7 +204,11 @@ def remove_weightnorm(model):
     waveglow = model
     for wavenet in waveglow.wavenet:
         wavenet.start = torch.nn.utils.remove_weight_norm(wavenet.start)
-        wavenet.in_layers = remove(wavenet.in_layers)
+        # SqueezeWave doesn't have weight norm in in_layers
+        try:
+            wavenet.in_layers = remove(wavenet.in_layers)
+        except ValueError:
+            pass
```

Comment:
I think a better check would be `if instance(model, SqueezeWaveModel)` as opposed to try, except

Reply:
i think this may cause circular import problems

Reply:
python will handle circular imports